target/postfix/main.cf

```diff
@@ -10,6 +10,8 @@ readme_directory = no
 alias_maps = hash:/etc/aliases
 alias_database = hash:/etc/aliases
 mydestination = $myhostname, localhost.$mydomain, localhost
+# Uncomment for container accounts to masquerade as the domain instead of the host
+# myorigin = $mydomain
```

Comment:
I am not quite sure about this one as it changes the default behaviour for everyone. Rather it should be made as a configuration whish as afaik already possible in this project.

Reply:
When I looked over this, I was not sure too. I think there already is an option to do this: Looking at the README, there's a line stating

"Note: If you want to use a bare domain (host name equals domain name) see FAQ."

I guess this change is not necessary and may possibly break things. I would rather like to remove it as well.

Reply:
It's commented, I put it there to make it easier for people (like me, which took some time) to find.

And it does break things.  And I was too lazy to document it.  Either way.

Reply:
I checked the sesction of the FAQ that you referenced.  That part of the FAQ is not about mail originating from the container host.  It's about situations where the MX record == the domainname.

Here is the relevent section of the postfix docs if you want to understand what I'm trying to do with this:

http://www.postfix.org/postconf.5.html#myorigin